examples/tutorials/Training_an_Exchange_Correlation_Functional_using_Deepchem.ipynb

```diff
@@ -7,78 +7,49 @@
    "source": [
```

Comment:
<p>The math doesn't render. I think this shuffled content is good, but as Bharath said keep the references.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>